analyzer/src/main/kotlin/managers/Pnpm.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2019-2020 HERE Europe B.V.
```

Comment:
Question: Is the copyright correct? Since the class is newly created, a date range starting with 2019 seems strange.

Reply:
I took _some_ inspiration for this class from https://github.com/oss-review-toolkit/ort/pull/2128. That's why I decided it would be appropriate to keep the copyright.